Interview/questions.md

```diff
@@ -28,7 +28,7 @@ Please keep in mind that this document is in a continuous state of flux, with ne
 - Are you familiar with `Design Patterns` and the `Gang of Four` book?
 - What is your opinion on the use of the `Singleton` pattern? What are its pros and cons?
 - Are you familiar with `Dependency Inversion`? If so, what is it good for?
-- Are you familiar with `Dependency Injection`? If so, what is it good for?
+- And what about the `inversion` principle?
```

Comment:
```suggestion
- And what about the `Dependency Inversion` principle?
```

Reply:
With this variant both lines 30 and 31 are talking about the in**v**ersion, let me correct the line 30.

Reply:
ah, right, missed the previous question